examples/lock-app/linux/include/LockManager.h

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <functional>
+
+#include <app/util/af.h>
+
+class LockManager
+{
+public:
+    void Init();
+
+    bool Lock(chip::Optional<chip::ByteSpan> pin);
+    bool Unlock(chip::Optional<chip::ByteSpan> pin);
+
+    static LockManager & Instance();
+    static LockManager instance;
```

Comment:
instance should not be public - the `Instance()` call is.

Reply:
Done